R/add_p.R

```diff
@@ -1,3 +1,14 @@
+#' Adds p-values to gtsummary table
```

Comment:
I love that this is a method function and works for tbl_cross!!

Is there any scenario we can think of that a user would want to use a custom p-value function and if so, should we support that?

Reply:
I think the API for custom p-values needs to change in the tbl_summary setting, so I am not keen to add the functionality here as well.  The way it's currently implemented, it's fragile and sometimes doesn't work.